Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'activemodel', '~> 4'
 gem 'activerecord', '~> 4'
 gem 'exception_helper'
 
-gem 'cassandra', :git => 'git@github.com:twitter/cassandra.git', :require => 'cassandra/1.2'
+gem 'twitter_cassandra', :git => 'git@github.com:backupify/twitter_cassandra.git', :branch => 'rename_root_namespace', :require => 'cassandra/1.2'
```

Comment:
once the gem PRs are merged, it will just point at master. It might be good practice to point at tags on master, but we don't usually do this.